AudioKit/Platforms/Common/CsoundObj.m

```diff
@@ -142,11 +172,15 @@ - (void)record:(NSString *)csdFilePath toURL:(NSURL *)outputURL
 
 - (void)record:(NSString *)csdFilePath toFile:(NSString *)outputFile
 {
+#ifdef TRAVIS_CI
+    [self performSelectorOnMainThread:@selector(runCsoundToDisk:) withObject:@[csdFilePath, outputFile] waitUntilDone:YES];
```

Comment:
Weird - there's probably some kind of race condition somewhere. We should get to the bottom of it...